backend/api/views/computetask.py

```diff
@@ -271,6 +275,32 @@ def get_serializer_class(self):
     def bulk_create(self, request, *args, **kwargs):
         return task_bulk_create(request)
 
+    @action(detail=True, url_name="input_assets")
+    def input_assets(self, request, pk):
+        input_assets = ComputeTaskInputAsset.objects.filter(task_input__task_id=pk).order_by("task_input__position")
```

Comment:
`output_assets` are ordered by `identifier` and `input_assets` by `position`. Is there an explanation for this difference ?

Reply:
Yes, ordering is mandatory. And outputs don't have `position` so I use `identifier` by default.

Reply:
To harmonize both views,  `input_assets` can be order also by `identifier` or for usage it's better to order them by `position` ?

Reply:
If I understand correctly, the purpose of position is to a deterministic order for inputs (we can have multiple inputs with the same identifier).

Reply:
I don't know why the design is not the same for outputs.

Reply:
Maybe to use position and stay in harmony, couldn't we order input by both, identifier first and then position ?

Reply:
@AurelienGasser @mblottiere what is your opinion on that?

Reply:
It's critical to returns inputs' position exactly as submitted by the datascientist (by identifier, ofc).
Output's order is not as critical since we don't support multiple outputs for now.

Alison's proposal LGTM.

Reply:
Changed in https://github.com/Substra/substra-backend/pull/496/commits/00ed11fbf5bd0b82b1ffb7daf51cac1bea25c61c